src/modules/admin/sections/ProfileEditor/ProfileEditor.tsx

```diff
@@ -37,9 +41,15 @@ export default function ProfileEditor() {
   }
 
   return (
-    <VStack align="flex-start">
-      <Text type="section.title">{t('appearance.profile.title')}</Text>
-      <EditableProfile profile={userData} />
-    </VStack>
+    <AdminSectionContainer heading="Profile Editor">
```

Comment:
This needs to be coming from the locales, and I think there's also a duplicate with this heading here.

Reply:
Updated with locales, and removed the duplication